visivo/models/row.py

```diff
@@ -7,6 +7,7 @@
 
 
 class HeightEnum(str, Enum):
```

Comment:
I wonder if we should also allow people to input pixels too if they want to be more specific about the height.

Reply:
I am having some layout issues, so I think I need to come back and fix them in a new PR.